apps/qubit/modules/object/actions/importAction.class.php

```diff
@@ -51,103 +51,30 @@ public function execute($request)
       $this->redirect($importSelectRoute);
     }
 
-    // set indexing preference
-    if (isset($request->noindex))
+    $options = array('noIndex' => ($request->getParameter('noindex') == 'on') ? false : true,
+                     'doCsvTransform' => ($request->getParameter('doCsvTransform') == 'on') ? true : false,
```

Comment:
can you just remove the part after ? since it'll evaluate as bool anyway? or is it a bit weird since you're doing the expression in the array definition like that?